README.md

```diff
@@ -11,20 +11,18 @@ Install and run it first.
 
 ### Build
 
-1. Install: git npm nodejs nasm libpng-dev dh-autoreconf
-2. run `npm install -g gulp`
+1. Install the following: `git`, `npm`, `nodejs`, `nasm`, `libpng-dev` and `dh-autoreconf`.
```

Comment:
@sudheesh001 That was missing in the previous readme guide. Which eventually made it to be splitted into different packages. Anyways this can be the better option.